js/table/actionBlock.ts

```diff
@@ -210,10 +210,11 @@ export class ActionBlock {
      */
     public gameFinished: KnockoutObservable<boolean>;
 
+    public suppressSetSitoutStatus: boolean;
```

Comment:
Why this change? Looks like this variable should belongs to TableView now ? Am I miss some other place where it is used?

Reply:
line number 469, should i move this function to tableview ?

Reply:
If it don't used anywhere in the `ActionBlock`, then please move it to `TableView`